history/taskana-simplehistory-provider/src/main/java/pro/taskana/simplehistory/impl/TaskanaHistoryEngineImpl.java

```diff
@@ -107,7 +112,27 @@ protected SqlSessionManager createSqlSessionManager() {
             DEFAULT, this.transactionFactory, taskanaEngineConfiguration.getDatasource());
     Configuration configuration = new Configuration(environment);
 
+    // set databaseId
+    String databaseProductName;
+    try (Connection con = taskanaEngineConfiguration.getDatasource().getConnection()) {
+      databaseProductName = DB.getDatabaseProductName(con);
+      String databaseProductId = DB.getDatabaseProductId(databaseProductName);
+      configuration.setDatabaseId(databaseProductId);
```

Comment:
should we create a method `DB#getDatabaseProductId(Connection)` so that usecases like this can be simplified?

Reply:
done